src/Project/Classes/Stores/Products/Product.java

```diff
@@ -0,0 +1,69 @@
+package Project.Classes.Stores.Products;
+
+import java.util.UUID;
+
+public abstract class Product {
+    private static int productNumber;
+    protected final UUID id;

```

Comment:
[Fix] si aici trebuie sa fie private

Reply:
Pai aici m-am gandit ca un produs poate sa fie fie instrument, fie accesoriu 
Si as avea un id unic pentru fiecare 
Si atributele din product sunt generale, atat pt instrument cat si pentru accesoriu
Nu sunt sigur daca am inteles bine :)))

Reply:
sa folosesti private, in loc de protected, asta am vrut sa zic